openshift_images/create-images.adoc

```diff
@@ -1,18 +1,12 @@
-:context: create-images
 [id="creating_images"]
```

Comment:
The anchor should match the context, which usually matches the title: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata